src/ducks/categories/CategoriesHeader.jsx

```diff
@@ -24,13 +24,14 @@ const CategoriesHeader = ({
 }) => {
   const globalCurrency = getGlobalCurrency(categories)
   const transactionsTotal = getTransactionsTotal(categories)
+  const [previousItem] = breadcrumbItems.slice(-2, 1)
```

Comment:
Huh ?

```
[1,2,3].slice(-2, 1)[0]
undefined
[0, 1,2,3].slice(-2, 1)[0]
undefined
```

What is this supposed to do ?

Is it not simpler to do index access directly ?

```
const previousItem = breadcrumbItems[breadcrumbItems.length - 2] ?
```